modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/engine/WhisperEngine.java

```diff
@@ -117,31 +113,22 @@ public String getEngineName() {
   @Activate
   @Modified
   public void activate(ComponentContext cc) {
-    logger.debug("Activated/Modified Whisper engine service class");
-    whisperExecutable = StringUtils.defaultIfBlank(
-        (String) cc.getProperties().get(WHISPER_EXECUTABLE_PATH_CONFIG_KEY), WHISPER_EXECUTABLE_DEFAULT_PATH);
+    var prop = cc.getProperties();
+    logger.debug("Activated/Modified Whisper engine service");
+    whisperExecutable = Objects.toString(prop.get(WHISPER_EXECUTABLE_PATH_CONFIG_KEY), WHISPER_EXECUTABLE_DEFAULT_PATH);
     logger.debug("Set Whisper path to {}", whisperExecutable);
 
-    whisperModel = StringUtils.defaultIfBlank(
-        (String) cc.getProperties().get(WHISPER_MODEL_CONFIG_KEY), WHISPER_MODEL_DEFAULT);
-    logger.debug("Whisper Language model set to {}", whisperModel);
+    whisperModel = Objects.toString(prop.get(WHISPER_MODEL_CONFIG_KEY), WHISPER_MODEL_DEFAULT);
+    logger.debug("Whisper model set to {}", whisperModel);
 
-    String t = (String) cc.getProperties().get(WHISPER_QUANTIZATION);
-    if (!StringUtils.isBlank(t)) {
-      quantization = Option.some(Quantizations.valueOf(t));
-    }
+    quantization = Objects.toString(prop.get(WHISPER_QUANTIZATION), null);
     logger.debug("Whisper quantization set to {}", quantization);
 
-    isVADEnabled = OsgiUtil.getOptCfgAsBoolean(cc.getProperties(), WHISPER_VAD);
-    logger.debug("Whisper Voice Activity Detection  set to {}", isVADEnabled);
+    isVADEnabled = OsgiUtil.getOptCfgAsBoolean(prop, WHISPER_VAD);
+    logger.debug("Whisper Voice Activity Detection set to {}", isVADEnabled);
```

Comment:
Not that this was different before… but it's fixed now. Instead of printing the `Opt` value, it is now printing the boolean value. So, now you should get something like:

> Whisper Voice Activity Detection set to true